conode/conode.go

```diff
@@ -49,6 +51,10 @@ const (
 
 var gitTag = ""
 
+// Seed the random number generator, because onet uses it to choose
+// permutations of rosters, etc.
+func init() { rand.Seed(time.Now().UTC().UnixNano()) }
```

Comment:
I agree that seeding with unixnano is not secure. An improvement would be to seed it with a cryptographically secure random source. But the security also depends on the pseudo random number generation algorithm in math/rand. Maybe seeding it correctly is enough because the permutation is not security critical and the security critical parts shouldn't be using math/rand anyway.

Reply:
Yes, it's not security critical. If you agree with me, ship it, please. Otherwise tell me why I'm misunderstanding where this is used, and then I'll seed it with cryptorand.

Reply:
It depends on what you want to protect against. If you want to protect against an attacker predicting which server you will contact, then I see two problems:

- using the go-random number method might be predictable (looking at a series of numbers might indicate what the next number will be)
- using `UnixNano` as seed makes it even more predictable

So if you want to avoid that an attacker can predict the sequence of requests, then don't use `rand` from go, but use `crypto/rand` in `Roster.RandomServerIdentity`.

Seeding with `UnixNano` seems better, but covers the underlying error. It's a non-sterilized band-aid. Looks good, but gives false security ;)

If you just want to avoid that the requests always start with the same nodes, then this might be OK. But it might give a false sense of security.

Reply:
This is a dark unexplored corner of the Byzantine fault tolerant threat model. Are the bad guys supposed to be able to target certain servers? Yes. Are they supposed to NOT get any advantage from predicting which server an honest client is going to talk to? Dunno.

This seems like a very abstract conversation to be having. I will just close this PR and instead make the roster selection cryptographically strong, since it's unlikely to be a performance problem.